serverless/knative_serving/serverless-applications.adoc

```diff
@@ -34,3 +34,11 @@ include::modules/kn-service-describe.adoc[leveloffset=+1]
 
 include::modules/verifying-serverless-app-deployment.adoc[leveloffset=+1]
 include::modules/interacting-serverless-apps-http2-gRPC.adoc[leveloffset=+1]
+
+== Using offline services
```

Comment:
IMO, the current title implies `offline service` feature, e.g. something supported from Serving side etc. Wdyt about this suggestion?

```suggestion
== Using `kn` in offline mode
```
Consequently the the subtitle should be "About offline mode" then,

Alternatively:
- "Using CLI in offline mode"
- "Using `kn` CLI in offline mode"

Reply:
Moreover, there's Serving's cluster-local feature, that doesn't expose the route outside of cluster. The `offline service` seems like something that might be easily mixed with that.

Reply:
Ahhh, in this case I think the CLI reference was actually the right place then, sorry @rh-max 
In the Jira we have it's just "offline services" so I did assume it was Serving related. Good call re changing the title @dsimansk

Reply:
@dsimansk Thanks, implemented the suggestion.

Reply:
@abrennan89 I think it actually fits here best. Notice that the section goes through the same `kn service` commands as the former sections in this assembly, only with the `--target` flag (currently without `kn service update` because of the bug). Plus it's not a reference, but a complete use case, fully relevant to the topic of creating Serverless applications.